packages/icon-button/src/IconButton.tsx

```diff
@@ -276,16 +276,17 @@ const IconButton = React.forwardRef((props: IconButtonProps, ref) => {
     if (isComponentType(child, 'Icon')) {
       const { size: childSize, title }: IconProps = child.props;
 
-      if (typeof title === 'string' && title.length > 0) {
-        return child;
-      }
+      const newChildProps: Partial<IconProps> = {
+        size: childSize || size,
```

Comment:
```suggestion
        size: childSize ?? size,
```

Instead?

Reply:
Hmm... I'm kinda okay here if `''` or `0` cause it to default to the IconButton's size, but I'm open to changing that if you feel strongly.

Reply:
Ah...wfm.